ViAn/GUI/mainwindow.cpp

```diff
@@ -232,25 +237,55 @@ void MainWindow::resizeEvent(QResizeEvent* event) {
  * @param newPos current position of the slider
  */
 void MainWindow::on_videoSlider_valueChanged(int newPos){
+    changeCounter++;
+    bool was_paused = false;
     // Make slider to follow the mouse directly and not by pageStep steps
+    if (abs(prev_slider_pos - newPos) == 1) {
+        return;
+    } else if (slider_moving) {
+        QPoint localMousePos = ui->videoSlider->mapFromGlobal(QCursor::pos());
+        float posRatio = localMousePos.x() / (float )ui->videoSlider->size().width();
+        int sliderRange = ui->videoSlider->maximum() - ui->videoSlider->minimum();
+        int sliderPosUnderMouse = ui->videoSlider->minimum() + sliderRange * posRatio;
+
+        /*
+        if (!mvideo_player->is_stopped() && !mvideo_player->is_paused()) {
+            std::cout << "pausing video" << std::endl;
+            emit set_pause_video();
+            was_paused = true;
+        }*/
+
+        std::chrono::milliseconds current_time = std::chrono::duration_cast<
+                std::chrono::milliseconds >(
+                    std::chrono::system_clock::now().time_since_epoch()
+        );
+        std::chrono::milliseconds time_since_last_slider_frame_update = current_time-slider_timer;
+        if (time_since_last_slider_frame_update.count() >= 200) {
```

Comment:
If 200 is a constant value, maybe it's better to add it as a const variable instead.